packages/lit-dev-content/site/docs/libraries/labs.md

```diff
@@ -6,14 +6,26 @@ eleventyNavigation:
+Lit Labs is an umbrella for Lit packages under development that we are actively seeking feedback on. While we encourage real-world use in order to help the feedback process, please note the following important aspects of Labs packages:
 
-Current Lit Labs projects include:
+- Lit Labs projects are published under the `@lit-labs` npm scope.
+- Breaking changes are likely to occur more frequently than in non-labs packages, but they will still respect standard semantic versioning guildelines and all changes will be published to the CHANGELOG files.
+- While bugs in non-lab packages are typically fixed with higher priority, we still treat labs bugs as high priority, and both are triaged as part of the same process.
+- When a Lit Labs project is ready to be graduated, it will begin publishing under the `@lit` scope—for example, the [`@lit/localize`](/docs/localization/overview/) package started out as a Lit Labs project. Once a package graduates, its first version under the `@lit` scope will match that of the latest in `@lit-labs` but only the `@lit` version will receive new updates.
+- We may decide to deprecate a Lit Labs project. In such cases, we wil notify the community and a deprecation warning will be added to the npm package. It will receive bug fix support for at least 6 months. A record of historical labs packages will be kept on this page.
 
-* [Motion](https://github.com/lit/lit/blob/main/packages/labs/motion/README.md#lit-labsmotion). Animation helpers for Lit templates. The `@lit-labs/motion` package includes an `animate` directive for adding tweening animations when an element changes state, as well as an `AnimateController` class for programmatically controlling and coordinating animations. You can find a number of motion examples in the [Playground](https://lit.dev/playground/#sample=examples/motion-simple).
-* [React](https://github.com/lit/lit/tree/main/packages/labs/react#lit-labsreact). React integration helpers for custom elements and reactive controllers.
-* [scoped-registry-mixin](https://github.com/lit/lit/tree/main/packages/labs/scoped-registry-mixin#lit-labsscoped-registry-mixin) Mixin for Lit that integrates with the speculative [Scoped CustomElementRegistry polyfill](https://github.com/webcomponents/polyfills/tree/master/packages/scoped-custom-element-registry).
-* [SSR](https://github.com/lit/lit/tree/main/packages/labs/ssr#lit-labsssr). A package for server-side rendering Lit templates and components.
-* [Task](https://github.com/lit/lit/blob/main/packages/labs/task/README.md#lit-labstask). A reactive controller for handling asynchronous tasks.
+Feedback is currently being solicited on the following Labs packages:
 
-When a Lit Labs project is ready for production it may be migrated to a new location outside of Lit Labs. For example, the [`@lit/localize`](/docs/localization/overview/) package started out as a Lit Labs project.
+| Package | Description | Docs | Feedback |
```

Comment:
I tried this out here: https://github.com/lit/lit.dev/pull/940

Reply:
Looking at the preview, I kind of like it, but I think a monochrome icon would look a lot better than the system emoji: https://pr940-d056ddc---lit-dev-5ftespv5na-uc.a.run.app/docs/libraries/labs/

Reply:
Stacking them in one column seems fine to save horizontal space, but I'd prefer the words to be written out, otherwise I think it's too obtuse:

> [📄 Docs](https://github.com/lit/lit/tree/main/packages/labs/motion#readme)
> [💬 Discussion](https://github.com/lit/lit/discussions)
> [🐞 File Issue](https://github.com/lit/lit/issues?q=is%3Aissue+is%3Aopen+%5Blabs%2Fmotion%5D)

Reply:
That works too, but I think reducing the font size for that column would help a lot

Reply:
btw, I don't think "File Issue" is very useful, _unless_ it pre-fills something specific to the labs package. If the link is the same for every package it doesn't deliver much.

Reply:
That was the idea. This fills in the title:

https://github.com/lit/lit/issues/new?template=bug-report.yaml&title=[labs/react]%20%3CBug%20summary%20here%3E

It seems like this might get it to fill in the drop-down based on [this](https://stackoverflow.com/questions/34146618/pre-populate-the-github-new-issue-form-using-the-querystring), but it doesn't work:

https://github.com/lit/lit/issues/new?template=bug-report.yaml&title=[labs/react]%20%3CBug%20summary%20here%3E&affected[]=React%20(@lit-labs/react)

Reply:
I tried some variations but unfortunately the pre-fill of the form doesn't seem to work for the multiselect, only text fields. :( I'll update the links at least fill the titles though

Reply:
Actually I like the link being a search with package name better. It would help prevent duplicates and I doesn't presume whether they want to file issue for a bug report or a feature request.